internals/generators/container/index.js

```diff
@@ -47,10 +47,22 @@ module.exports = {
   }],
   actions: (data) => {
     // Generate index.js and index.test.js
+    let componentTemplate;
```

Comment:
What I don't understand is that this code is [doing the same thing that the component generator](https://github.com/react-boilerplate/react-boilerplate/blob/master/internals/generators/component/index.js#L39) has been doing already. Why fail here and not there?